contracts/ExampleToken.cdc

```diff
@@ -58,21 +45,20 @@ pub contract ExampleToken: FungibleToken {
     /// out of thin air. A special Minter resource needs to be defined to mint
     /// new tokens.
     ///
-    pub resource Vault: FungibleToken.Provider, FungibleToken.Receiver, FungibleToken.Balance {
+    pub resource Vault: FungibleToken.Provider, FungibleToken.Receiver, FungibleToken.Balance, MetadataViews.Resolver {
 
         /// The total balance of this vault
         pub var balance: UFix64
 
-        // initialize the balance at resource creation time
+        ///Do we need extra fields for metadata?
```

Comment:
Yes, we should add extra fields for the display view

Reply:
Do you mean including in the example token contract fields for its symbol or description?

Or do you mean to include the `MetadataViews.Display` view as one of the views that the vault resolves?